packages/formik-date/package.json.lerna_backup

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
This file shouldn't be here. Not sure if you exited the `postinstall` step while it was running?